model/pipeline/src/main/proto/metrics.proto

```diff
@@ -204,6 +211,10 @@ message MonitoringInfoUrns {
     TOTAL_MSECS = 5
         [(org.apache.beam.model.pipeline.v1.beam_urn) =
              "beam:metric:ptransform_execution_time:total_msecs:v1"];
+
+    USER_DISTRUBUTION_COUNTER_URN_PREFIX = 6
```

Comment:
why are we distinguishing `USER_` things here? I am really not convinced that there should be sharing between user and system-defined metrics in this way.

System-defined metrics should be proto field _not_ enums or strings attached to a generic metric mechanism.

User-defined metrics should be fairly opaque and a totally different abstraction.